spec/mailers/user_mailer_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+
+RSpec.describe UserMailer, :type => :mailer do
+  it 'sends an email when a review is added' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.